package.json

```diff
@@ -1,36 +1,27 @@
 {
-  "name": "electron-react-boilerplate",
-  "description": "A foundation for scalable desktop apps",
-  "keywords": [
-    "electron",
-    "boilerplate",
-    "react",
-    "typescript",
-    "ts",
-    "sass",
-    "webpack",
-    "hot",
-    "reload"
-  ],
-  "homepage": "https://github.com/electron-react-boilerplate/electron-react-boilerplate#readme",
-  "bugs": {
-    "url": "https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues"
-  },
+  "name": "draft-craft",
```

Comment:
I believe naming it 'Draft-Craft' would be better, it will ensure uniformity across the app.

Reply:
The regex for the `name` property doesn't allow for that casing. Hence, was forced to use lower case here.

Reply:
> The regex for the `name` property doesn't allow for that casing. Hence, was forced to use lower case here.

alright